tomviz/ModuleSlice.cxx

```diff
@@ -317,11 +328,16 @@ bool ModuleSlice::serialize(pugi::xml_node& ns) const
   pointNode.append_attribute("y").set_value(point[1]);
   pointNode.append_attribute("z").set_value(point[2]);
 
+  // Map colors
+  pugi::xml_node mapScalars = ns.append_child("MapScalars");
+  mapScalars.append_attribute("value").set_value(m_widget->GetMapScalars() ==
+                                                 1);
+
   // Let the superclass do its thing
   return Module::serialize(ns);
 }
 
-bool ModuleSlice::deserialize(const pugi::xml_node& ns)
+bool ModuleSlice::deserialize(const pugi::xml_node& ns) /**/
```

Comment:
What is the /**/ for? It looks like a mistake.

Reply:
Hmm, not sure how that got there. Removed.